include/aspect/utilities.h

```diff
@@ -156,6 +156,11 @@ namespace aspect
     bool fexists(const std::string &filename);
 
     /**
+     * Retrieve the last line of a file
+     */
+    std::string get_last_line(std::ifstream *in);
```

Comment:
But an `ifstream` *is* an `istream`. Why unnecessarily restrict what kinds of arguments you can pass to this function?